tool/tctl/common/sessions_command.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/api/types/events"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service"
+)
+
+// SessionsCommand implements "tctl sessions" group of commands.
+type SessionsCommand struct {
+	config *service.Config
+
+	// format is the output format (text, json, or yaml)
+	format string
+	// sessionsList implements the "tctl sessions ls" subcommand.
+	sessionsList *kingpin.CmdClause
+	// FromUTC is the start time to use for the range of sessions listed by the recorded session listing command
+	FromUTC string
+
+	// ToUTC is the start time to use for the range of sessions listed by the recorded session listing command
+	ToUTC string
+}
+
+// Initialize allows SessionsCommand to plug itself into the CLI parser
+func (c *SessionsCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	c.config = config
+
+	sessions := app.Command("sessions", "Operate on recorded sessions.")
+	c.sessionsList = sessions.Command("ls", "List recorded sessions.")
+	c.sessionsList.Flag("format", "Output format, 'text', 'json', or 'yaml'").Default(teleport.Text).StringVar(&c.format)
```

Comment:
Nit: Do we have a list of supported formats somewhere in a variable so we don't have to enumerate them here? In tsh we have `defaultFormats`, can we move it some commonplace it can be reused from?